appveyor.yml

```diff
@@ -19,7 +19,8 @@ for:
     cache:
       - ./vendor/bundle
     install:
-      - rvm use 2.6.3
```

Comment:
I think Ubuntu should remain at 2.6 since we already cover 2.7 on OSX. Not having to install a newer version of Ruby also keeps the job faster. :fast_forward:

Reply:
👍